.storybook/preview.js

```diff
@@ -3,6 +3,7 @@ import '../src/styles/index.css';
 
 export const parameters = {
   actions: { argTypesRegex: '^on[A-Z].*' },
+  docs: { source: { excludeDecorators: true } },
```

Comment:
This isn't yet documented ... I had to dig through a PR to figure out how to make it work...